gobblin-data-management/src/main/java/gobblin/data/management/retention/DatasetCleaner.java

```diff
@@ -100,9 +106,12 @@ public DatasetCleaner(FileSystem fs, Properties props) throws IOException {
             Optional.of("Dataset-cleaner-pool-%d")));
     this.service = MoreExecutors.listeningDecorator(executor);
 
+    List<Tag<?>> tags = Lists.newArrayList();
+    tags.addAll(Tag.fromMap(AzkabanTags.getAzkabanTags()));
     // TODO -- Remove the dependency on gobblin-core after new Gobblin Metrics does not depend on gobblin-core.
-    this.metricContext = this.closer.register(Instrumented.getMetricContext(new State(props), DatasetCleaner.class));
+    this.metricContext = this.closer.register(Instrumented.getMetricContext(new State(props), DatasetCleaner.class, tags));
```

Comment:
Just a comment: you are not gaining anything by calling `Instrumented.getMetricContext` instead of simple calling `MetricContext.builder....build()`. However, it was like that before, so you can leave it as is.